start_exp272.sh

```diff
@@ -1,25 +1,125 @@
-#!/bin/sh
+#!/bin/bash
+
+# Default system library path
+OS_LIB_PATH="/lib"
+
+# Spacecraft file system paths
+EXP_HOME_PATH="/home/exp272"
+EXP_DOOM_BIN_NAME="opssat-doom"
+EXP_DOOM_BIN_PATH="${EXP_HOME_PATH}/src/bin/${EXP_DOOM_BIN_NAME}"
+EXP_DOOM_EXEC=$EXP_DOOM_BIN_PATH
+
+# If "dev" argument is provided then use Docker file system paths
+# TODO: Prepare the dev environment with /lib/ld-linux-armhf.so.3 symnolic link
+if [ "$1" = "dev" ]; then
+  source /home/user/poky_sdk/environment-setup-cortexa8hf-neon-poky-linux-gnueabi
+  OS_LIB_PATH="/home/user/poky_sdk/tmp/sysroots/beaglebone/lib"
+  EXP_HOME_PATH="/home/user/opssat-doom"
+  EXP_DOOM_BIN_PATH="${EXP_HOME_PATH}/src/bin/${EXP_DOOM_BIN_NAME}"
+  EXP_DOOM_EXEC="qemu-arm ${EXP_DOOM_BIN_PATH}"
+fi
+
+# Experiment directory paths
+EXP_LIB_PATH="${EXP_HOME_PATH}/lib"
+EXP_TOGROUND_PATH="${EXP_HOME_PATH}/toGround"
+EXP_DEMOS_PATH="${EXP_HOME_PATH}/demos"
 
 # Define the list of demos
 demos="e1m7-607 impfight m1-fast m1-normal m1-simple"
```

Comment:
If you're in sync with main, you should run the `zero-...` demos, since these use the vanilla rng table.

Reply:
Made a flag for this in case we revert to the previous version of the build.